src/CourseService.php

```diff
@@ -386,6 +386,89 @@ public function UpdateAssetsFromUploadedFile($courseId, $uploadLocation)
     	
     	return $response;
     }
+
+    /// <summary>
+    /// Import a SCORM .pif (zip file) from the local filesystem using async client.
+    /// </summary>
+    /// <param name="courseId">Unique Identifier for this course.</param>
+    /// <param name="absoluteFilePathToZip">Full path to the .zip file</param>
+    /// <returns>List of Import Results</returns>
+    public function ImportCourseAsync($courseId, $absoluteFilePathToZip)
```

Comment:
typehint the response

Reply:
typehint is disallowed due the version of php. Would you like me to upgrade?